rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -1503,6 +1511,23 @@ TensorList * MasterGraph::mask_meta_data()
     return &_mask_tensor_list;
 }
 
+TensorList * MasterGraph::pixelwise_meta_data()
```

Comment:
This API can be removed and mask_meta_data() can be used for both polygons and pixel wise masks right?

Reply:
Sure